chart/hooks/pre-upgrade/remove-deployment-updater-readiness.sh

```diff
@@ -18,8 +18,18 @@ spec:
       - name: cc-deployment-updater-cc-deployment-updater
         readinessProbe: ~
 '
+set +o pipefail
+scheduler_list=$(kubectl get statefulsets --namespace "$NAMESPACE" | grep scheduler | cut -d " " -f 1)
+set -o pipefail
 
-kubectl patch statefulset --namespace "$NAMESPACE" scheduler --patch "$patch"
+if [ "${scheduler_list}" == "" ]; then
+  echo "No scheduler statefulset found."
+  exit 0
+fi
+
+for i in ${scheduler_list}; do
+  kubectl patch statefulset --namespace "$NAMESPACE" $i --patch "$patch"
```

Comment:
Please always run `make lint` before submitting a PR (it will not pass CI if it fails the `lint` step).

```
In ./chart/hooks/pre-upgrade/remove-deployment-updater-readiness.sh line 31:
  kubectl patch statefulset --namespace "$NAMESPACE" $i --patch "$patch"
                                                     ^-- SC2086: Double quote to prevent globbing and word splitting.
```

In addition we generally put variables in curly braces as well (although that is not enforced by shellcheck, so we often miss it). The final style would ideally be:

```
for scheduler in ${scheduler_list}; do
  kubectl patch statefulset --namespace "${NAMESPACE}" "${scheduler}" --patch "${patch}"
done
```

Reply:
@jandubois, thanks for good advice, I've updated the code.